pallets/creditcoin/src/weights.rs

```diff
@@ -446,4 +446,10 @@ impl<T: frame_system::Config> crate::WeightInfo for WeightInfo<T> {
 			.saturating_add(Weight::from_parts(0, 0))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+
+	fn register_address_v2() -> Weight {
+		Weight::from_parts(0,0)
```

Comment:
Needs benchmarks in `benchmarking.rs` as well in order to get this weight correctly benchmarked